src/EventSubscriber/TranslationOverviewRequestSubscriber.php

```diff
@@ -83,13 +94,12 @@ public function onRequest(GetResponseEvent $event): void {
     }
 
     $entity_type = $matches[1];
-    if (!$this->entityTypeManager->getDefinition($entity_type)) {
+    if (!$this->entityTypeManager->hasDefinition($entity_type)) {
```

Comment:
Shall we replace deprecated class `GetResponseEvent` with `RequestEvent` (the `$event` parameter type)?

Reply:
Yes, thanks! I also included a test for this redirect which I had deleted.